euphonic/force_constants.py

```diff
@@ -296,7 +296,7 @@ def calculate_qpoint_phonon_modes(
 
         .. math::
 
-          \sum_{\\kappa^\\prime \\alpha^\\prime}D_{\\alpha {\\alpha}^\\prime}^{\\kappa {\\kappa}^\\prime}(\\mathbf{q}) {e}_{\\mathbf{q}\\nu{\\kappa^\\prime}{\\alpha^\\prime}} =
+          \\sum_{\\kappa^\\prime \\alpha^\\prime}D_{\\alpha {\\alpha}^\\prime}^{\\kappa {\\kappa}^\\prime}(\\mathbf{q}) {e}_{\\mathbf{q}\\nu{\\kappa^\\prime}{\\alpha^\\prime}} =

```

Comment:
Are you sure these double-escapes are needed? I think `.. math::` blocks automatically handle them. In the RST documentation the examples use single backslashes. https://docutils.sourceforge.io/docs/ref/rst/mathematics.html

Was there a rendering problem under some conditions?

Reply:
There were no specific rendering issues, but when compiling the docs locally I was getting escape warnings e.g. `\s not recognised as an escape sequence` and if there are a lot of them that could hide other useful warnings

Reply:
That's weird, on Master branch with Linux and sphinx-build 3.5.4 I don't get any warning related to this. (It mostly complains about underline lengths in the changelog!)

Reply:
Oh wait my mistake, the warnings are actually in the test outputs! When running `pytest -v tests_and_analysis\test\euphonic_test\test_force_constants.py` (Python 3.7 on Windows)

Reply:
I don't see it in the Windows CI logs though :thinking:

Reply:
Adding to the confusion, multi-line strings seem to do some automatic escaping anyway:

```
In [100]: """ok there
     ...: is a break in
     ...: here. Or two. \s pls no complain"""
Out[100]: 'ok there\nis a break in\nhere. Or two. \\s pls no complain'
```

So it's not clear how pytest would ever see an invalid string to complain about.

Reply:
Ok looking at this https://stackoverflow.com/questions/46543194/does-r-stand-for-something-in-sphinx it looks like we have a few options:
* Leave everything double escaped (easiest option, a bit ugly but otherwise no harm done)
* Single escape everything, double escape things like `\r` (and anything else we find)
* Single escape everything, use raw string e.g. `r'this is a raw string'`. This seems to work but I'm not sure if it will give us issues elsewhere.

Reply:
Raw string seems the nicest solution from the source code end. But I'm a bit confused as to why this would be system-specific, or if it is needed at all.

Maybe in some environments `\s` is interpreted as "space" and that inconsistency causes it to skip a stage of automatic double-escape?

I suppose we can keep the double-escapes in the PR for now, and migrate slowly to raw strings as things are added or maintained.